calibrations/sepd/eventplane/scripts/runProd.py

```
@@ -0,0 +1,872 @@
+            try:
+                # We open in 'errors=ignore' to avoid crashing on weird binary garbage
+                with open(log_file, "r", encoding="utf-8", errors="ignore") as f:
+                    content = f.read()
+
+                    # Check for success
+                    if success_pattern.search(content):
+                        finished_files.add(log_file)
+                        current_success_count += 1
+
+                    # Optional: Add check for "Abnormal termination" here to fail early
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)